### ISSUE TITLE
[Schedule Overrides] When add two overrides for same show only one displays

### ISSUE THREAD

**Issue by tonyzeoli** [netmix/radio-station#298]
**Describe the bug**

See: [](https://wordpress.org/support/topic/when-2-overrides-used-for-one-show-only-one-displays/#post-13820385)

"As an example, if we have a normal show 10am – 2pm and on a certain week this changes to 10am – 1pm and then a 1pm – 2pm show, it only shows the 1pm-2pm override and leaves the original show details from 10am-1pm."

**To Reproduce**

Set an override for two back to back shows minus one hour.

If Show A is at 10 am to 2 pm, set override to 10 am to 1 pm.
Then, set the next Show B override from 1 pm to 2pm.
Will only display the override for the B and not A.

**Expected behavior**

Both shows should display overrides.

**Screenshots**

No screenshot available.

**Additional context**

N/A

**Comment by majick777** [comment]
I'm unclear hear are you adding two overrides? Overrides can't override each other, they need to be specific to a time and date. So an override from 12-2pm and another for the same date at 1-2pm is not desirable, it's a user input conflict really... I guess we could add conflict checking to Overrides when they are being saved similar to how we do so for Shows? ie. Detect for conflicts and warn the user and disable the Override.

**Comment by majick777** [comment]
@tonyzeoli I see you are referencing a WordPress support forum ticket:
https://wordpress.org/support/topic/when-2-overrides-used-for-one-show-only-one-displays/
If you could include that reference in future it would be helpful. I do think this is probably a duplicate of #290 
and we should wait for the user to confirm whether 2.3.3.7 has fixed it and if so we could repurpose this ticket for Overide conflict checking on save.

**Comment by tonyzeoli** [comment]
Oh, weird. The link didn't post after the word, "See:" with the colon at the end in the original description. Not sure what I did wrong there, but you caught up with it anyway. Yes, that's the post from that user. I don't know that the OP was saying that they were the same show, but could certainly be. Now that I reread it, that makes sense.  He should just schedule the show again from 1 pm to 2 pm for that off week and then remove it the following. Yeah, if you override a show on a given day, you should only be able to override that show only once and have that override display. Then, fill in the empty slot with a new Show time.

**Comment by majick777** [comment]
Closing this out, as support issue was resolved.
